packages/@aws-cdk/aws-codebuild/README.md

```diff
@@ -313,6 +314,22 @@ new codebuild.Project(this, 'Project', {
 })
 ```
 
+## Certificate
```

Comment:
Let's add this to an existing code example, with a comment explaining what it represents, and noting this is an optional property - I don't think it deserves an entire section.